zuul.ansible.d/nodesets.yaml

```diff
@@ -625,6 +625,16 @@
         nodes:
           - fedora-35
 
+- nodeset:
```

Comment:
I would prefer we don't had yet another `controller-pythonXY` node. This is a bit pointless since they all point on Fedora 35 BTW :-).